### ISSUE TITLE
Preserve scroll position

### ISSUE THREAD

**Issue by rahilwazir** [Soundnode/soundnode-app#663]
Is this possible to remember the last scroll position of the view?

I have a playlist with more than 150+ tracks, its annoying that you have to always scroll down to the end.

- **Soundnode-app**
    `0.6.2`

- **OS**
    Ubuntu 64bit

- **Steps to reproduce**

  1. Select any playlist which can trigger scroll
  2. Scroll down to the end
  3. Now navigate within the app somewhere
  4. Come back and view the playlist again the scroll starts from top.

**Comment by dgattey** [comment]
Yeah, I would love this. Just looked into implementing it and realized it's going to be a bit of a headache. First of all, it's also an issue on the main stream page, and anywhere else where there's a scrollable view that contains dynamic content. 

We don't only need to save the scroll position, but also all the web content that has been loaded in via requests to the Soundnode API. We can save a JS object on pushing a page to history, but it's pretty tough to do for how much content we have. But someone who's been working with the app for longer may have a better idea about how to implement.

**Comment by Chriz007** [comment]
I've also been asking this, but sadly my suggestions have been dismissed.

**Comment by weblancaster** [comment]
Let's have this as a nice to have and add to the roadmap.. right now let's focus on the major issues.
